orderbook/src/main.rs

```diff
@@ -187,6 +192,7 @@ async fn main() {
             orderbook.clone(),
             Arc::new(database.clone()),
             Arc::new(event_updater),
+            Arc::new(vault_updater),
```

Comment:
This shouldn't be added the the list of service maintainers because of the outstanding `todo!()` which would cause panics in staging.

Reply:
Yes. it is in the above checklist to remove before merge. This was only for my local testing to ensure the events were being streamed.